card/mod/core/set/all/collection.rb

```diff
@@ -228,10 +228,11 @@ def edit_fields
   end
 
   def normalized_edit_fields
-    edit_fields.map do |name, options|
-      options ||= Card.quick_fetch(name).name
+    edit_fields.map do |name_or_card, options|
+      next [name_or_card, options || {}] if name_or_card.is_a?(Card)
```

Comment:
So, here's what I'm thinking we can say (and implement and document):

There are 3 ways to create a web form on Wagn:

1. passively.  Allow Wagn to autogenerate a form from the read content/structure
2. simple configuration.  via either:
 - voo.edit_fields (what we're currently doing with edit_structure.  has to be simple objects. creates a simple linear form).  Nest syntax version may need to be a simple comma separated list?
 - voo.edit_structure (like voo.structure, gives a card name.  This way you can really customize the layout of your form)
3. advanced (coder) configuration
- override `def edit_fields` (and do something to signal to wagn that we should use it)
- override something else to structure the whole form

Basic ideas:
1. voo stuff needs to stay simple, because every voo value has to be short and easily cached
2. we've got to make the coder API coder friendly.

Thoughts?